config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -1437,405 +1242,58 @@
       "yBucketNumber": null,
       "yBucketSize": null
     },
-
     {
-      "id": 26,
+      "id": 29,
       "gridPos": {
         "h": 8,
         "w": 12,
         "x": 12,
-        "y": 26
+        "y": 66
       },
-      "type": "graph",
-      "title": "Work Queue Depth",
-      "thresholds": [],
+      "type": "heatmap",
+      "title": "Latency of Release Creation in Seconds",
       "pluginVersion": "9.1.6",
+      "maxDataPoints": 25,
+      "description": "Measure release creation latency from end of testing to release created ",
       "legend": {
-        "avg": false,
-        "current": false,
-        "max": false,
-        "min": false,
-        "show": true,
-        "total": false,
-        "values": false
+        "show": false
       },
-      "aliasColors": {},
-      "dashLength": 10,
-      "fill": 1,
-      "lines": true,
-      "linewidth": 1,
-      "nullPointMode": "null",
-      "options": {
-        "alertThreshold": true
+      "cards": {
+        "cardPadding": null,
+        "cardRound": null
       },
-      "pointradius": 2,
-      "renderer": "flot",
-      "seriesOverrides": [],
-      "spaceLength": 10,
+      "color": {
+        "cardColor": "#b4ff00",
+        "colorScale": "sqrt",
+        "colorScheme": "interpolateOranges",
+        "exponent": 0.5,
+        "mode": "spectrum"
+      },
+      "dataFormat": "tsbuckets",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "hideFrom": {
+              "tooltip": false,
+              "viz": false,
+              "legend": false
+            }
+          }
+        },
+        "overrides": []
+      },
+      "heatmap": {},
+      "hideZeroBuckets": true,
+      "highlightCards": true,
+      "reverseYBuckets": false,
       "targets": [
         {
           "exemplar": true,
-          "expr": "workqueue_depth{name=\"snapshot\", namespace=\"integration-service\"}",
-          "instant": false,
-          "interval": "",
-          "legendFormat": "{{name}}",
-          "refId": "A",
-          "editorMode": "builder"
-        },
-        {
-          "refId": "B",
-          "hide": false,
-          "editorMode": "builder",
-          "expr": "workqueue_depth{name=\"integrationtestscenario\", namespace=\"integration-service\"}",
```

Comment:
Why this metric is removed?

Reply:
cause it contains block with [snapshotenvironmentbinding](https://github.com/redhat-appstudio/integration-service/blob/242306f761202358d3ce4c219deaa1c90adaa241/config/grafana/dashboards/integration-service-dashboard.json#L1493C6-L1500C10) , should I have just remove the specific block only ?

Reply:
yes, keep still valid metrics there